src/main/java/net/neoforged/neoforge/client/RenderTypeGroup.java

```diff
@@ -16,7 +16,6 @@
 public record RenderTypeGroup(RenderType block, RenderType entity, RenderType entityFabulous) {
 
     public static RenderTypeGroup EMPTY = new RenderTypeGroup(null, null, null);
-
```

Comment:
This is strange - I guess the formatter doesn't like the addition to the canonical constructor.